i18next-scanner.config.js

```diff
@@ -0,0 +1,61 @@
+// This file is part of React-Invenio-Forms
+// Copyright (C) 2021 Graz University of Technology.
+//
+// React-Invenio-Deposit is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+const fs = require('fs');
+const chalk = require('chalk');
+
+module.exports = {
+    options: {
+        debug: true,
+        // removeUnusedKeys: true, // removing unused keys
+        browserLanguageDetection: true,
+        func: {
+            list: ['i18next.t', 'i18n.t', 't'],
+            extensions: ['.js', '.jsx']
+        },
+        trans: false,
+        lngs: ['en','de'],
+        ns: [
+            // file name (.json)
+            'translations',
+        ],
+        defaultLng: 'en',
+        defaultNs: 'translations',
+        resource: {
+            // The path where resources get loaded from. Relative to current working directory. 
+            loadPath: 'src/lib/translations/{{lng}}/{{ns}}.json',
+
+            // The path to store resources. Relative to the path specified by `gulp.dest(path)`.
+            savePath: 'src/lib/translations/{{lng}}/{{ns}}.json',
+
+            jsonIndent: 2,
+            lineEnding: '\n'
+        },
+        nsSeparator: false, // namespace separator
+
+        //Set to false to disable key separator
+        // if you prefer having keys as the fallback for translation (e.g. gettext). 
+        keySeparator: false,
+    },
+    transform: function customTransform(file, enc, done) {
+        "use strict";
+        const parser = this.parser;
+        const content = fs.readFileSync(file.path, enc);
+        let count = 0;
+
+        parser.parseFuncFromString(content, { list: ['i18next.t', 'i18n.t', 't'] }, (key, options) => {
```

Comment:
Is that list the same as defined above? If so, maybe we should define it as a const and reuse it here?